pages.de/common/nix.md

```diff
@@ -0,0 +1,24 @@
+# nix
+
+> Dienstprogramme für die Nix-Sprache und den Nix-Speicher.
+> Weitere Informationen: <https://nixos.org>.
+
+- Suche nach einem Paket über seinen Namen oder seine Beschreibung:
+
+`nix search {{search_term}}`
+
+- Starten einer Nix-Shell, die die angegebenen Pakete zur Verfügung stellt:
+
+`nix run {{nixpkgs.pkg1 nixpkgs.pkg2 nixpkgs.pkg3...}}`
```

Comment:
The following fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/nix.md) page is updated:

```suggestion
`nix run {{nixpkgs.pkg1 nixpkgs.pkg2 ...}}`
```

Wrong syntax is used for zero or more arguments.